lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -535,7 +535,7 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
             if (renderer is CanvasKitRenderer) {
               assert(
                 decoded.arguments is int,
-                'Argument to Skia.setResourceCacheMaxBytes must be an int, but was ${decoded.arguments.runtimeType}',
+                'Argument to Skia.setResourceCacheMaxBytes must be an int, but was ${(decoded.arguments as Object?).runtimeType}',
```

Comment:
I wonder why casting `decoded.arguments as Object?` doesn't make you access `runtimeType` trough `?.`... I guess null also has `runtimeType` so it doesn't matter?

Reply:
Yeah, `null` has all the `Object` methods, like `runtimeType`, `toString`, `operator==`, `hashCode`, etc. It's also surprising why `dynamic.runtimeType` is considered to be a dynamic invocation. Everything has a `runtimeType` in Dart. But I guess maybe `dynamic.anything` is considered "bad" by the lint.

Reply:
This is a bug in the lint

Reply:
Filed https://github.com/dart-lang/sdk/issues/55957.